data/author.json

```diff
@@ -1,6 +1,15 @@
 [
   {
-    "id": "Fabien BERNARD",
+    "id": "truongsinh",
+    "name": "TruongSinh Tran-Nguyen",
+    "bio": "Head of Tech @ Smarp.",
+    "avatar": "avatars/alien-male-icon.png",
```

Comment:
Found! You've forgetten to add this avatar, so I/netlify can't build properly the website